pkg/component/observability/logging/fluentcustomresources/cluster_parsers_test.go

```diff
@@ -38,7 +38,7 @@ var _ = Describe("Logging", func() {
 								TimeFormat: "%Y-%m-%dT%H:%M:%S.%L%z",
 								TimeKeep:   ptr.To(true),
 							},
-							Decoders: []fluentbitv1alpha2.Decorder{
+							Decoders: []fluentbitv1alpha2.Decoder{
```

Comment:
Will this work? It's called `Decorder` in the upstream package.

Reply:
This was by mistake, fixed.